scenarioo-client/app/manage/generalSettings/generalSettings.controller.js

```diff
@@ -43,6 +44,10 @@ function GeneralSettingsController($scope, BranchesResource, ConfigService, Vers
             }
         );
 
+        DiffViewerConfigResource.get(function onSuccess(result) {
```

Comment:
one other option could also be, to even refactor the config endpoint to deliver all this state (configuration object and all other status info) in one call, But this could still be done, when really needed. Two calls are allright for me.